metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/StringFunctions.java

```diff
@@ -343,4 +343,89 @@ public Object apply(List<Object> args) {
       return String.format(format, formatArgs);
     }
   }
+
+  @Stellar( name="CHOP"
```

Comment:
IMHO the `chop` function would be helpful when fine-tuning string extraction operations through stellar (E.g. when extracting specific variations of hostnames or URLs). 

Aside of that, I went ahead and added the `chomp` string function as well since I agree that it would be of use.